### PR TITLE
create_admin command (fixes #841)

### LINKED ISSUE #841: Administrator configuration
We need to create 2 separate admins:
* one with access to Django Admin (it also need to have UserProfile, let's make it `hello@codeforpoznan.pl`)
* one with administrator priviledges, but without access to Django Admin (it will be `wolontariat@wrk.org.pl`)

We need also to make fabric create these users on `fab install` and that it will use `secrets.py` for passwords.

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

**Comment by magul** [comment]
There's a problem, when user already exists:
```
backend_1   | Traceback (most recent call last):
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/backends/utils.py", line 65, in execute
backend_1   |     return self.cursor.execute(sql, params)
backend_1   | psycopg2.IntegrityError: duplicate key value violates unique constraint "auth_user_username_key"
backend_1   | DETAIL:  Key (username)=(hi@codeforpoznan.pl) already exists.
backend_1   | 
backend_1   | 
backend_1   | The above exception was the direct cause of the following exception:
backend_1   | 
backend_1   | Traceback (most recent call last):
backend_1   |   File "manage.py", line 10, in <module>
backend_1   |     execute_from_command_line(sys.argv)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
backend_1   |     utility.execute()
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 356, in execute
backend_1   |     self.fetch_command(subcommand).run_from_argv(self.argv)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 283, in run_from_argv
backend_1   |     self.execute(*args, **cmd_options)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 330, in execute
backend_1   |     output = self.handle(*args, **options)
backend_1   |   File "/volontulo/backend/apps/volontulo/management/commands/create_admin.py", line 31, in handle
backend_1   |     is_active=True,
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/contrib/auth/models.py", line 159, in create_user
backend_1   |     return self._create_user(username, email, password, **extra_fields)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/contrib/auth/models.py", line 153, in _create_user
backend_1   |     user.save(using=self._db)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/contrib/auth/base_user.py", line 80, in save
backend_1   |     super(AbstractBaseUser, self).save(*args, **kwargs)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/base.py", line 807, in save
backend_1   |     force_update=force_update, update_fields=update_fields)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/base.py", line 837, in save_base
backend_1   |     updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/base.py", line 923, in _save_table
backend_1   |     result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/base.py", line 962, in _do_insert
backend_1   |     using=using, raw=raw)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/manager.py", line 85, in manager_method
backend_1   |     return getattr(self.get_queryset(), name)(*args, **kwargs)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/query.py", line 1076, in _insert
backend_1   |     return query.get_compiler(using=using).execute_sql(return_id)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/models/sql/compiler.py", line 1107, in execute_sql
backend_1   |     cursor.execute(sql, params)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/backends/utils.py", line 80, in execute
backend_1   |     return super(CursorDebugWrapper, self).execute(sql, params)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/raven/contrib/django/client.py", line 123, in execute
backend_1   |     return real_execute(self, sql, params)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/backends/utils.py", line 65, in execute
backend_1   |     return self.cursor.execute(sql, params)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/utils.py", line 94, in __exit__
backend_1   |     six.reraise(dj_exc_type, dj_exc_value, traceback)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/utils/six.py", line 685, in reraise
backend_1   |     raise value.with_traceback(tb)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/django/db/backends/utils.py", line 65, in execute
backend_1   |     return self.cursor.execute(sql, params)
backend_1   | django.db.utils.IntegrityError: duplicate key value violates unique constraint "auth_user_username_key"
backend_1   | DETAIL:  Key (username)=(hi@codeforpoznan.pl) already exists.
```
I'm working on it!

**Comment by magul** [comment]
Problem solved @w1stler

File: README.md [hunk 1]
```diff
@@ -19,8 +19,11 @@ docker-compose up
 and point your browser to [http://localhost:8000](http://localhost:8000) and [http://localhost:4200](http://localhost:4200)
 
 ## Initial admin credentials
- * **user**: admin@volontuloapp.org
- * **pass**: stx123
+We have 2 levels of admin users and 2 initial users - with and without Django Admin access:
+username               | password | Django Admin access
+---------------------- | -------- | -------------------
+hi@codeforpoznan.pl    | cfp123   | yes
```
Comment (w1stler) [inline README.md]: I am using hello alias

Comment (magul) [reply README.md]: We have only hello... I will fix it.

**Comment by magul** [comment]
@w1stler `hello@codeforpoznan.pl` it is.

**Pull request merged**